models/balancer/arbitrum/balancer_v2_arbitrum_transfers_bpt.sql

```diff
@@ -0,0 +1,45 @@
+{{
+    config(
+        alias='transfers_bpt',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'evt_tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["stefenon"]\') }}'
+    ) 
+}}
+
+{% set event_signature = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef' %}
+{% set project_start_date = '2021-08-26' %}
+
+WITH registered_pools AS (
+    SELECT
+      DISTINCT poolAddress AS pool_address
+    FROM
+      {{ source('balancer_v2_arbitrum', 'Vault_evt_PoolRegistered') }}
+  )
```

Comment:
Since this CTE will query the full table every time, do you think we need to apply any incremental filter here?

Reply:
Since the result table of the query is already incremental, I think we can apply the same filter to this table. I just changed that in the latest commit.

Reply:
@Hosuke I added the incremental filter, but the test failed due to it. Is there a way to fix it keeping the filter?

Reply:
The problem is in the config block, I will point out in the next comment.